### PR TITLE
misc(build): report to buildtracker on commit via ghactions

### PR DESCRIPTION
**Pull request by paulirish**
I've set up https://buildtracker.dev/ and backfilled it with the last 18months of commits:
https://lh-build-tracker.herokuapp.com/builds/limit/1000

image

The UX takes some clicking around to get used to but it's pretty powerful. I like this better than the [alternatives](https://github.com/GoogleChrome/lighthouse/issues/10472#issue-582635603) mostly for the rich UI of exploring historical commits.

This PR adds a github action workflow that will report to buildtracker on every PR push and master commit. (I considered using the buildtracker action, but it breaks [on `push`](https://github.com/paularmstrong/build-tracker-action/issues/2) and currently [always](https://github.com/paularmstrong/build-tracker-action/issues/3) posts a PR comment). By default buildtracker only plots master branch but all the data is there in case we wanted to plot other branches. 

Here's an interesting find from our recent commits... 21K increase in dt-bundle. It's  attributable to the `update axe-core to 3.5.1` commit, natch. [2-commit comparison link](https://lh-build-tracker.herokuapp.com/builds/54408d241ebd8937874777077e522a6ad6ce59bb/a5ff4f51abf430950cde2a53e9cee085d908c6de?sizeKey=stat&disabledArtifactsVisible=true&graphType=AREA&comparedRevisions=54408d241ebd8937874777077e522a6ad6ce59bb&comparedRevisions=a5ff4f51abf430950cde2a53e9cee085d908c6de&activeArtifacts=dist%2Flighthouse-dt-bundle.js)
image

Followups:
 - Build tracker [supports budgets](https://buildtracker.dev/docs/budgets/) (and some nice reporting [options](https://buildtracker.dev/docs/guides/advanced-ci#reporting-budget-results)), however we can add that on later after the data collection has begun.
 - Remove bundlesize

ref #10472

### CODE REVIEW

File: .github/workflows/buildtracker.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - run: yarn --frozen-lockfile
+    - run: yarn build-all
+
+    - name: Store in buildtracker
```
Comment (connorjclark) [inline .github/workflows/buildtracker.yml:28]: wdyt of just adding this to the basic workflow? You need to install node, run yarn and build for this to work, so it's already a heavy workflow.

#10549 reused basic.yml for the same reason.

File: .github/workflows/buildtracker.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      run: yarn bt-cli upload-build
+      env:
+        # https://buildtracker.dev/docs/guides/github-actions#configuration
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (connorjclark) [inline .github/workflows/buildtracker.yml:32]: I think `GITHUB_TOKEN` is already a set env variable.

File: .github/workflows/buildtracker.yml (hunk 1 above)
Comment (paulirish) [reply .github/workflows/buildtracker.yml]: good call.

File: .github/workflows/buildtracker.yml (hunk 2 above)
Comment (paulirish) [reply .github/workflows/buildtracker.yml]: good call.

File: .github/workflows/buildtracker.yml (hunk 1 above)
Comment (paulirish) [reply .github/workflows/buildtracker.yml]: > why is installing node 10 so heavy?  

dont know.. i see it taking a while today but in [previous master commits](https://github.com/GoogleChrome/lighthouse/actions?query=workflow%3A%F0%9F%92%A1%F0%9F%8F%A0+branch%3Amaster), that step takes <3s.

**Comment by paulirish** [comment]
`git config pull.rebase true`. what was i thinking? never again. 😶

File: package.json [hunk 3]
```diff
@@ -74,6 +74,7 @@
     "static-server": "node lighthouse-cli/test/fixtures/static-server.js"
   },
   "devDependencies": {
+    "@build-tracker/cli": "^1.0.0-beta.14",
```
Comment (connorjclark) [inline package.json:77]: theres beta 15 now https://github.com/paularmstrong/build-tracker/compare/v1.0.0-beta.14...v1.0.0-beta.15

Comment (paulirish) [reply package.json:77]: good catch. i actually asked him to publish that one. :)
it fixed the slow responsiveness of the hoverline on https://lh-build-tracker.herokuapp.com/

**Comment by paularmstrong** [comment]
Really awesome to see this coming together!

> The UX takes some clicking around to get used to but it's pretty powerful.

If there's anything that you think could make things easier, would love to see feature suggestions!

**Pull request merged**